pkg/sql/parser/col_types.go

```diff
@@ -514,6 +514,9 @@ func DatumTypeToColumnType(t Type) (ColumnType, error) {
 			}
 			return arrayOf(elemTyp, Exprs(nil))
 		}
+		if typ, ok := t.(tOidWrapper); ok {
```

Comment:
heh, looks like the entire `default` clause and this new arm can be a `switch t.(type) {`